src/locales/fr.json

```
@@ -18,7 +18,17 @@
           "line": {
             "1": "Votre Cozy sera supprimé",
             "2": "Vous ne pourrez plus accéder à votre Cozy",
-            "3": "L'ensemble de vos données seront supprimées"
+            "3": "L'ensemble de vos données seront supprimées",
+            "4": "Veuillez saisir votre mot de passe pour valider cette action :"
```

nit: We should use a non-breaking space here
```suggestion
            "4": "Veuillez saisir votre mot de passe pour valider cette action :"
```